index.html

```diff
@@ -290,6 +290,28 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <footer>
+      <div class="form-div">
+        <div class="form-info">
+          <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <form action="https://formspree.io/f/form_id" method="post">
+        <div class="name-wrapper">
+          <div class="input-wrapper first-name-input">
+            <input type="text" id="first-name" placeholder="First Name">
+          </div>
+          <div class="input-wrapper last-name-input">
+            <input type="text" placeholder="Last Name">
+          </div>
+        </div>
+        <div class="email-wrapper">
+          <input type="email" placeholder="Your email">
+        </div>
+        <textarea cols="30" rows="10" placeholder="Your message here..."></textarea>
+        <button type="button" class="btn-get">Get in touch</button>
+      </form>
+      </div>
+    </footer>
```

Comment:
- [Optional] kindly Add the social links at the footer as show below just as the Figma design .
![Figma design](https://user-images.githubusercontent.com/93598189/184005468-ad9ba60a-71c1-48b1-a991-1a602a44b114.PNG)